lumicks/pylake/population/dwelltime.py

```diff
@@ -480,3 +480,55 @@ def _exponential_mle_optimize(
 
     # output parameters as [amplitudes, lifetimes], -log_likelihood
     return result.x, -result.fun
+
+
+def _dwellcounts_from_statepath(statepath, exclude_ambiguous_dwells=True):
+    """Calculate the dwell counts and slicing indices for all states in a state path trajectory.
+
+    Note: the counts are the number of frames or time points. To convert to proper
+    dwelltimes, multiply the counts by the time step between points.
+
+    Parameters
+    ----------
+    statepath : numpy.typing.ArrayLike
+        Time-ordered array of state labels
+    exclude_ambiguous_dwells : bool
+            Determines whether to exclude dwelltimes which are not exactly determined. If `True`, tracks which
+            start in the first frame or end in the last frame of the kymograph are not used in the analysis,
+            since the exact start/stop times of the binding event are not definitively known.
```

Comment:
Copy/paste? Kymograph doesn't seem relevant here? Also, the indent is not correct.